rust/relay/server/src/ebpf/linux.rs

```diff
@@ -17,6 +17,9 @@ use crate::{AllocationPort, ClientSocket, PeerSocket};
 
 pub struct Program {
     ebpf: aya::Ebpf,
+
+    #[expect(dead_code, reason = "We are just keeping it alive.")]
+    stats: AsyncPerfEventArray<MapData>,
```

Comment:
I don't think this should be necessary because we are keeping around a copy in all tasks as well.